src/mmw/js/src/draw/views.js

```diff
@@ -245,11 +253,16 @@ var PlaceMarkerView = Marionette.ItemView.extend({
 
     ui: {
         'items': '[data-shape-type]',
-        'button': '#delineate-shape',
+        'button': '#delineate-shape'
     },
 
     events: {
-        'click @ui.items': 'onItemClicked',
+        'click @ui.items': 'onItemClicked'
+    },
+
+    onShow: function() {
+        // TODO: the viewport setting doesn't appear to be working.
```

Comment:
It might be easier to tell if this is working if it's set on the predefined shapes selector. It looks like it may be working for this one, but it's hard to say because there's not a lot of text.